### ISSUE TITLE
[Enhancement] New option to save the parts as separate xhtml files

### ISSUE THREAD

**Issue by terabyte-gc** [gvellut/jncep#8]
I see there is already a flag to save the images separately in the output folder:
`-i, --images Flag to indicate that the images of the novel should be extracted into the output folder`

Would it be possible to add a new flag to save also the parts in separate xhtml files?

I  sometimes want to find specific occurrences of a character, name, scene, or something else in all volumes and right now my "solution" is to unpack the epub files and then run a grep on the whole folder. With the option to save separate xhtml files from the epub it would be easier.

**Comment by gvellut** [comment]
Thank you for your feedback.

OK I will add a flag for next version (I will release this week-end probably).

**Comment by terabyte-gc** [comment]
Thanks for considering it :)

**Comment by gvellut** [comment]
I have made a new release (0.21). There is now a `-c` /  `--content` switch in the `epub` and `update` commands in order to extract the raw HTML. You can upgrade with:

`pip install --upgrade jncep `

**Comment by terabyte-gc** [comment]
Hmm, it doesn't seem to be working. This is what I was using before as a test, and it generates 3 images and 1 epub file:
`jncep epub --byvolume --images https://j-novel.club/s/culinary-chronicles-of-the-court-flower/`

After updating to 0.21 I switched to this line instead:
`jncep epub --byvolume --content --images https://j-novel.club/s/culinary-chronicles-of-the-court-flower/`

The output however includes 3 images, 1 **empty** html file for part 1 (out of 2 parts), and **no epub** file.

**Comment by gvellut** [comment]
The command you wrote gives me the expected result on my end.

What OS and Python version are you using? 
Can you post the output log?

**Comment by terabyte-gc** [comment]
Windows 10 (build 1909) / Python 3.9.2

I tried to use powershell instead of command prompt (I'm using that python command from a .bat file) and I safely got the output. I see what the problem is now (encoding troubles again):
```
Login with email 'XXX'...
Fetching metadata for 'culinary-chronicles-of-the-court-flower'...
Fetching part 'Culinary Chronicles of the Court Flower: Volume 1 Part 1'...
Fetching images found in part content...
Image 1...
Image 2...
Fetching part 'Culinary Chronicles of the Court Flower: Volume 1 Part 2'...
Some Unicode characters unlikely to be readable with the base fonts of an EPUB reader have been replaced
Fetching images found in part content...
Image 1...
Fetching cover image...
Extracting images...
Extracting content...
Logout...
*** An unrecoverable error occured ***
'charmap' codec can't encode character '\u014c' in position 625: character maps to <undefined>
```

**Comment by terabyte-gc** [comment]
I also tried using a different series just in case: https://j-novel.club/s/kobold-king. I got a similar error though:
```
Login with email 'XXX'...
Fetching metadata for 'kobold-king'...
Fetching part 'Kobold King: Volume 1 Part 1'...
Fetching images found in part content...
Image 1...
Fetching part 'Kobold King: Volume 1 Part 2'...
Fetching part 'Kobold King: Volume 1 Part 3'...
Fetching part 'Kobold King: Volume 1 Part 4'...
Fetching part 'Kobold King: Volume 1 Part 5'...
Fetching part 'Kobold King: Volume 1 Part 6'...
Fetching part 'Kobold King: Volume 1 Part 7'...
Fetching part 'Kobold King: Volume 1 Part 8'...
Fetching cover image...
Extracting images...
Extracting content...
Logout...
*** An unrecoverable error occured ***
'charmap' codec can't encode character '\u2012' in position 19339: character maps to <undefined>
```

**Comment by gvellut** [comment]
OK. I should have set an explicit encoding... It should be fixed in version 0.22 that I just uploaded.

**Comment by terabyte-gc** [comment]
> OK. I should have set an explicit encoding... It should be fixed in version 0.22 that I just uploaded.

Yes, the new 0.22 version seems to be working as expect. I tried with both series mentioned above and no errors this time.

P.S.: Just a reminder that README.md needs to be updated with the new `--content` flag ;)

**Comment by gvellut** [comment]
OK Great. I will update the documentation soon.
